config/routes.rb

```diff
@@ -34,6 +34,7 @@
   resource :feedback, controller: 'feedback'
 
   namespace :editorial do
+    resources :assets, only: [:index, :new, :create]
```

Comment:
I thought so, but as it turns out convention says if the controller exists that's all that's required.